organization/_posts/2020/2020-01-30-coordination.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "HSF Weekly Meeting #179, 30 January, 2019"
+layout: meetings
+---
+
+# {{page.title}}
+
+Present/Contributors: Graeme Stewart, Witek Pokorski, Caterina Doglioni, David Lange, Efe Yazgan, Mason Proffitt, Andrea Valassi, Philippe Canal, Stefan Roiser, Kyle Knoepfel, Eduardo Rodrigues, Ben Morgan, Teng Jian Khoo, Andrea Rizzi, Danilo Piparo, Liz Sexton-Kennedy, Gloria Corti, Attila Krasznahorkay
+
+Apologies: Agnieszka Dziurda, Pere Mato, Serhan Mete
+
+## News, general matters
+
+- Letters of Collaboration and Support
+    - Signed off on 3 of these at the beginning of the week
+    - *Please try to make such requests in a timely fashion - ideally 2 weeks before any deadlines*
+
+- ESCAPE Software Catalogue, <https://projectescape.eu/escape-catalogue>
+    - Caterina and Graeme met with Kay Graf from the ESCAPE project
+    - They have a deliverable to provide a "software catalogue" for the project
+        - "ESCAPE services will be part of the global EOSC catalogue of scientific software, to ensure global open access and long-term usability."
```

Comment:
Spell out in parentheses what EOSC means, as I'm sure almost nobody knows - I did not.